pkg/handlers/index/list_test.go

```diff
@@ -16,12 +16,14 @@
 package index
 
 import (
+	"encoding/json"
```

Comment:
we use `github.com/goccy/go-json` instead of `encoding/json`

Reply:
ok